ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,24 @@
+### **Description**
```

Comment:
We already pushed issue template to git, please catch up with develop and remove this from changes.